### DISCUSSION TITLE
Does OpenSSL support signature Algorithm ECDSA-With-SHA-1?

### DISCUSSION THREAD

**Discussion by 3quanfeng** [openssl/openssl#22429] [Q&A]
We have OCSP response signed using algo ECDSA-With-SHA-1. Our OCSP client, when verify the response from the provider, doesn't seem to like it and throw "SSL: error:0D0C5006:asn1 encoding routines:ASN1_item_verify:EVP lib error:27069075:OCSP routines:OCSP_basic_verify:signature failure".
I would like to confirm the version of openssl started to not support signature algorithm ECDSA-With-SHA-1?

**Comment by levitte** [comment]
We do, but with its [RFC 3279](https://www.rfc-editor.org/rfc/rfc3279#section-2.2.3) name, which is just a teeny bit different: ecdsa-with-SHA1

**Reply by levitte** [reply]
(casing should make no difference, but that extra dash between "SHA" and "1" does)
